core/modules/system/css/system.theme.css

```diff
@@ -429,37 +429,46 @@ ul.secondary a.active {
  */
 .messages {
   position: relative;
+  box-sizing: border-box;
   margin: 6px 0;
-  padding: 10px 10px 10px 55px; /* LTR */
+  padding: .9em .625em 1em 3.438em; /* LTR */
   border-radius: 4px;
-  min-height: 32px;
+  min-height: 3.2em;
   overflow: hidden;
 }
 [dir="rtl"] .messages {
-  padding: 10px 55px 10px 10px;
+  padding: .625em 3.438em .625em .625em;
 }
 .messages:before {
   content: '';
   position: absolute;
   top: 0;
   left: 0; /* LTR */
-  width: 38px;
+  width: 2.375em;
   height: 100%;
-  background-position: center 10px;
+  background-position: center .625em;
   background-repeat: no-repeat;
-  background-size: 24px;
+  background-size: 1.5em;
+}
+.messages :nth-child(2) {
+  margin-top: 0;
+  padding-top: 0;
+}
+.messages :last-child {
+  margin-bottom: 0;
+  padding-bottom: 0;
```

Comment:
This isn't for any HTML that's in theme_status_message, it's for the content in the message. We don't want large amounts of white space at the top or bottom of the message, because it looks dumb. This gets around it.
The problem is I don't know what HTML will be inside of the status message. It's *probably* just p, ul and ol, but this is a better catch all.